assets/static/filters.js

```diff
@@ -64,12 +68,65 @@ function setPause() {
     $(selectors.icon).removeClass("fa-circle-o-notch fa-spin fa-search").addClass("fa-pause");
 }
 
+function renderHistory() {
+    const storage = window.localStorage;
```

Comment:
Is it possible to pass localStorage here, that way you can use your mock without having to override `window.localStorage`?